Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM nginx:1.17.7-alpine
 
+# Override default configuration files
+COPY ./config/nginx.conf /etc/nginx/nginx.conf
```

Comment:
Correct me if I'm wrong but my understanding of the need for these files is for demonstrative purposes only, right? Because almost anyone who would use this image would need to add their own.

Reply:
No, these are added on purpose to ensure that by default this container is runnable as a non-privileged user.

Specifically, the changes from port `80` to `8080` and the removal of the `user` directive are effected by overriding these files.

Reply:
Noted.